### DISCUSSION TITLE
What OS are you running netatalk on?

### DISCUSSION THREAD

**Discussion by rdmark** [Netatalk/netatalk#435] [Polls]
We're curious to hear from you the community which OS you're running netatalk on!
Having some insights here will help us prioritize feature development, bug fixes and testing.

Please comment below with any additional details about your particular setup!
We would love to know more.

**Comment by ghost** [comment]
Not sure if we need Mac OS X server as all versions until 10.9 had a built-in AFP file server so netatalk would be redundant in these cases. Otherwise looking good, should capture most use cases.

**Reply by rdmark** [reply]
The one reason would be for AFP over AppleTalk, no? I've had a conversation recently with someone working on the MacPorts netatalk package who talked about this use case.

Anyways, I agree that it's too niche to call out in the list. If someone's doing this I'd love to hear about it in the comments here though. :)

**Comment by bunk3m** [comment]
Your survey only allows one choice.  At the moment running Netatalk on Centos & FreeBSD.  Both used equally.

**Reply by rdmark** [reply]
FWIW I tried to make the poll multi-choice, but Github Discussions only support radio buttons it seems (unless I overlooked something obvious). My workaround was to put the "*primarily*" qualifier in the poll heading. :)

Thanks for sharing your setup!

**Comment by rdmark** [comment]
One person told me in private: Debian, FreeBSD and OmniOS

**Comment by Michael-Wohlstadter** [comment]
Primarily CentOS at the moment (which is how I voted).  I'm attempting to compile for Solaris and in the not too distant future the CentOS box will be replaced with an Ubuntu box.

**Comment by pierrepaap** [comment]
No too many voters I see, since my vote accounts for 3% :-D

I was trying to move to SMB since I read somewhere that it was the way apple was going, but the performance are so abysmal without quite a bit of tweaking (and even with, still pretty bad)

**Reply by rdmark** [reply]
@pierrepaap Thanks for voting! I think 32 votes to date is pretty respectable all things considered. :)

When you say SMB, I assume you mean Samba running on an open source OS? I’m curious if you have ran benchmarks for various scenarios?

**Reply by pierrepaap** [reply]
You assume correctly. Samba on Arch linux (same file server)

I didn't run benchmarks since the file browsing was absolutely terrible, probably in the minute range. While this I managed to improve with some tweaks (although still slower than netatalk), opening a 5MB picture with Preview remained painfully slow.
Even with Netatalk it's not lightning speed, despite having good network (that I didn't benchmark either but we're only a few home users with wifi 5 and 1Gb wired for the server) but it's an order of magnitude or 2 better
